### DISCUSSION TITLE
How do I change font color for a particular block of text?

### DISCUSSION THREAD

**Discussion by soumikghosh23** [mmistakes/minimal-mistakes#2696] [Q&A] [answered]

**Comment by mmistakes** [comment, marked as answer]
There is no easy way to do this with Markdown in Jekyll unless you want to write inline CSS or use HTML.
I suggest you search around [Jekyll's help forums](https://talk.jekyllrb.com/) as this isn't something the theme can help with and is more of a Markdown/Jekyll issue.

Closing for this reason.
